CHANGELOG.md

```diff
@@ -42,7 +45,12 @@ Version <dev> introduces Stripe instrumentation, allows the agent to record addi
     - 'green$'
   ```
 
+- **Bugfix: Update Elasticsearch datastore instance metric to use port instead of path**
+
+  Previously, the Elasticsearch datastore instance metric (`Datastore/instance/Elasticsearch/<host>/*`) used the path as the final value. This caused a [metrics grouping issue](https://docs.newrelic.com/docs/new-relic-solutions/solve-common-issues/troubleshooting/metric-grouping-issues) for some users, as every document ID created a unique metric. Now, the datastore instance metric has been updated to use the port as the final value. This also has the benefit of being more accurate for datastore instance metrics, as this port is directly associated with the already listed host.
```

Comment:
```suggestion
  When we initially released Elasticsearch instrumentation, the Elasticsearch datastore instance metric (`Datastore/instance/Elasticsearch/<host>/*`) used the request path as the final value. This follows a pattern we use for some datastore instance metrics. After observing more Elasticsearch datastore usage, we've determined port is a more accurate value for these metrics. Going forward, the metric will use the port as the final value instead of the path.
```